design/ceph/ceph-config-via-cluster-crd.md

```diff
@@ -0,0 +1,53 @@
+# Ceph Config via Ceph Cluster CRD
+
+## Goal
+
+Adding an addition simpler way to configure the Ceph config options via the CephCluster CRD.
+The goal is not to replace the `rook-config-override` ConfigMap yet, but to reduce it's use to almost zeor for 99% of the cases.
```

Comment:
```suggestion
The goal is not to replace the `rook-config-override` ConfigMap since it is still needed in some scenarios such as if mons need settings applied at first launch.
```